go.mod

```diff
@@ -58,7 +58,8 @@ replace (
 	github.com/contiv/libOpenflow => github.com/everoute/libOpenflow v0.0.0-20210716071814-4ef09249fae5
 	github.com/contiv/libovsdb => github.com/everoute/libovsdb v0.0.0-20210326110222-6c508538aa65
 	//github.com/contiv/ofnet => github.com/everoute/ofnet v0.0.0-20211229083707-5cb1e875a38b
-	github.com/contiv/ofnet => github.com/echkenluo/ofnet v0.0.0-20220316073105-f24ad3ca4c99
+	//github.com/contiv/ofnet => github.com/echkenluo/ofnet v0.0.0-20220316073105-f24ad3ca4c99
+	github.com/contiv/ofnet => github.com/echkenluo/ofnet v0.0.0-20220329095940-b05eed62ea96
```

Comment:
This is my personal repo, we need to pick this commit to public repo in everoute dir.

Reply:
has been modified